index.src.html

```diff
@@ -1168,10 +1223,11 @@ <h4 algorithm id="collectfromcredentialstore-passwordcredential">
   </ol>
 
   <h4 algorithm id="create-passwordcredential">
-    `PasswordCredential`'s `[[Create]](options)`
+    `PasswordCredential`'s `[[Create]](origin, options)`
   </h4>
 
-  <dfn for="PasswordCredential" method>\[[Create]](options)</dfn> is called with a
+  <dfn for="PasswordCredential" method>\[[Create]](origin, options)</dfn> is called with
+  an [=environment settings object/origin=] (|origin|) and a
```

Comment:
This should link to https://html.spec.whatwg.org/multipage/origin.html#concept-origin with [=origin=] rather than https://html.spec.whatwg.org/multipage/webappapis.html#concept-settings-object-origin.

Reply:
ok.